src/khoj/utils/helpers.py

```diff
@@ -318,6 +321,7 @@ class ConversationCommand(str, Enum):
     ConversationCommand.Image: "Generate images by describing your imagination in words.",
     ConversationCommand.Automation: "Automatically run your query at a specified time or interval.",
     ConversationCommand.Help: "Get help with how to use or setup Khoj from the documentation",
+    ConversationCommand.Summarize: "Create an appropriate summary using provided documents.",
```

Comment:
Any idea why the /summarize related diffs are showing up in this PR to add support for docx?

Reply:
this is due to rebasing so that tester can be added. gonna convert this PR to a draft because there is some more work to be done.